modules/aurora-postgres/ssm.tf

```
@@ -1,5 +1,5 @@
 locals {
-  ssm_path_prefix = format("/%s/%s", var.ssm_path_prefix, module.cluster.id)
+  ssm_path_prefix = length(var.ssm_cluster_name_override) > 0  ? format("/%s/%s", var.ssm_path_prefix, var.ssm_cluster_name_override) : format("/%s/%s", var.ssm_path_prefix, module.cluster.id)
```

```suggestion
  ssm_path_prefix = format(var.ssm_path_prefix_format, module.cluster.id)
```

Remove variable `ssm_path_prefix`
Add variable `ssm_path_prefix_format` with default value `"/aurora-postgres/%s"`

Than you will be able to override it

---

or to support backcompatibility

```
ssm_path_prefix = format(var.ssm_path_prefix_format, var.ssm_path_prefix, module.cluster.id)
```

Add variable ssm_path_prefix_format with default value "/%s/%s"

---

this will still append module.cluster.id? so instead of the cluster_id that might look like `namespace+tenant+environment+service_name-main` , we wanted a shorter parameter store name, so if ssm_cluster_name_override, we'll use that instead.

---

you can set `ssm_path_prefix_format="/aurora-postgres/short-cluster-name"`
and that would be the same